tests/ert_tests/ert3/console/integration/test_cli.py

```diff
@@ -666,6 +681,75 @@ def test_cli_local_test_run_specific_realization(tmpdir):
             assert pathlib.Path(f"local-test-run-{run_id}/output.json").exists()
 
 
+def test_cli_local_test_run_storageinput(tmpdir):
+    with chdir(tmpdir):
+        with Storage.start_server():
+            args = ["ert3", "init", "--example", "polynomial"]
+            with patch.object(sys, "argv", args):
+                ert3.console._console._main()
+
+            os.chdir("polynomial")
+
+            for tainted_repo_dir in glob.glob("local-test-run*"):
+                # Because the runpath for local test runs is unpredictable,
+                # we need this in order to reliably guess where to look.
+                shutil.rmtree(tainted_repo_dir)
+
+            # Load values for parameters directly into storage
+            designed_coeffs_path = "experiments/doe/coefficients_record.json"
+
+            realization_to_run = 1
+            ensemble_size = 3
+
+            # Overwrite the designed coefficients for test runtime reasons:
+            pathlib.Path(designed_coeffs_path).write_text(
+                # The values in here are also used to assert that the correct
+                # realization number was executed.
+                json.dumps(
+                    [{"a": iens, "b": iens, "c": iens} for iens in range(ensemble_size)]
+                ),
+                encoding="utf-8",
+            )
+
+            # Modify the ensemble size in the example to avoid making 1000 http
+            # requests for the record uploading in this test:
+            ens_config = yaml.safe_load(
+                pathlib.Path("experiments/doe/ensemble.yml").read_text(encoding="utf-8")
+            )
+            ens_config["size"] = ensemble_size
+            pathlib.Path("experiments/doe/ensemble.yml").write_text(
+                yaml.dump(ens_config), encoding="utf-8"
+            )
+            args = [
```

Comment:
Why this needs to be loaded via loaded via `ert3 record load` and just by cfg file?